crates/vizia_core/src/state/binding.rs

```diff
@@ -4,6 +4,30 @@ use std::collections::{HashMap, HashSet};
 use crate::prelude::*;
 use crate::state::{BasicStore, LensCache, ModelOrView, Store, StoreId};
 
+/// A view on a binding to allow for better debugging
+struct BindingView<L: Lens> {
+    lens: L,
+}
+
+impl<L: Lens> View for BindingView<L> {
+    fn build<F>(self, cx: &mut Context, _content: F) -> Handle<Self> {
+        let id = cx.entity_manager.create();
+        let current = cx.current();
+        cx.tree.add(id, current).expect("Failed to add to tree");
+        cx.cache.add(id).expect("Failed to add to cache");
+        cx.style.add(id);
+        cx.views.insert(id, Box::new(self));
+
+        Handle { entity: id, p: Default::default(), cx }
+    }
+
+    fn element(&self) -> Option<&'static str> {
+        self.lens.name()
+    }
+
+    fn draw(&self, _cx: &mut DrawContext, _canvas: &mut Canvas) {}
+}
+
```

Comment:
I don't think we should have a dedicated view for bindings. We had that before but it adds superfluous information, such as a bounding box and clip region which are meaningless for bindings.

It should be possible to get the lens name from the binding if you add a method to `BindingHandler`, because all of the bindings are stored in hashmap in context with the entity as the key.

Reply:
Ok